diffusion/evaluation/clean_fid_eval.py

```diff
@@ -62,6 +63,7 @@ def __init__(self,
                  batch_size: int = 16,
                  image_key: str = 'image',
                  caption_key: str = 'caption',
+                 load_strict_model_weights: bool = True,
```

Comment:
Isn't this already specified by the "trainer"?  Can we reuse those settings?

Reply:
hm can you elaborate on how to do this?